docs/localInstallation.md

```diff
@@ -1,12 +1,12 @@
-# Phis Deployment
+PostgreSQL# OpenSILEX Deployment
```

Comment:
did you mean "# OpenSILEX Deployment" ? without PostgreSQL

Reply:
yes that is what I meant